docs/References.bib

```diff
@@ -297,6 +297,40 @@ @article{Casoni2012
   year    = 2013
 }
 
+@article{CerdaDuran2007,
+  author =       {{Cerd{\'a}-Dur{\'a}n}, P. and {Font}, J.~A. and {Dimmelmeier},
+                  H.},
+  title =        "General relativistic simulations of
+                  passive-magneto-rotational core collapse with microphysics",
+  journal =      {\aap},
```

Comment:
fix (either supply the journal or add whatever file is missing that defines `\aap`